src/Test/Utilities/Shared/Diagnostics/TrackingDiagnosticAnalyzer.cs

```diff
@@ -75,7 +75,11 @@ public void VerifyAllAnalyzerMembersWereCalled()
 
         public void VerifyAnalyzeSymbolCalledForAllSymbolKinds()
         {
-            var expectedSymbolKinds = new[] { SymbolKind.Event, SymbolKind.Field, SymbolKind.Method, SymbolKind.NamedType, SymbolKind.Namespace, SymbolKind.Property };
+            var expectedSymbolKinds = new[] 
+            {
+                SymbolKind.Event, SymbolKind.Field, SymbolKind.Local, SymbolKind.Method, SymbolKind.NamedType, SymbolKind.Namespace, SymbolKind.Parameter, SymbolKind.Property
```

Comment:
Is this enough test coverage or do I need more specific tests? I don't know how the testing works in this area.